lib/delayed/backend/worker.rb

```diff
@@ -16,7 +16,7 @@ def configure
                       end
         self.delay = config.delay_seconds || 0
         self.timeout = config.visibility_timeout || 5.minutes
-        self.expires_in = config.message_retention_period || 4.days
+        self.expires_in = config.message_retention_period || 345600 # 4 days
```

Comment:
There were odd errors about `4.days` not being able to be coerced into a Fixnum. This was not an issue previously. The answer wasn't obvious within a few minutes, and it isn't important enough to spent more time debugging.